TODO.md

```diff
@@ -8,8 +8,8 @@
     * `void libevdev_set_log_function(libevdev_log_func_t logfunc, void *data);`
 * libevdev_device_log_func_t
     * `void libevdev_set_device_log_function(struct libevdev *dev,
-				      libevdev_device_log_func_t logfunc,
-				      enum libevdev_log_priority priority,
-				      void *data);`
+                      libevdev_device_log_func_t logfunc,
```

Comment:
It seems like most of this repo uses spaces, so I changed some tabs to spaces

Reply:
Are you requesting that I revert the tab changes, or requesting that I do it differently next time?

Reply:
Do it differently next time.